print.php

```diff
@@ -29,7 +29,7 @@
       $triageVenDis = $row['TriageVenDis'];
       $weight = $row['Weight'];
       $temperature = $row['Temperature'];
-      $bloodPressure = $row['BloodPressure'];
+      $bloodPressure = $row['Systolic'] . "/" . $row['Diastolic'];
       $glucose = $row['Glucose'];
       $heartRate = $row['HeartRate'];
       $lastPeriod = $row['LastPeriod'];
```

Comment:
A number of these are throwing undefined index. These notices will be hidden on production site.